scripts/falco-driver-loader

```diff
@@ -618,9 +630,9 @@ if [[ -z "$MAX_RMMOD_WAIT" ]]; then
 	MAX_RMMOD_WAIT=60
 fi
 
-DRIVER_VERSION="@DRIVER_VERSION@"
+DRIVER_VERSION=${DRIVER_VERSION:-"@DRIVER_VERSION@"}
```

Comment:
I am not sure if this is something we want!
@leogr

Reply:
Ok for `DRIVER_VERSION`, but I'd avoid making `FALCO_VERSION` (see my comment below).